homework1/homework1.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="homework1.css">
+    <title>Homework1</title>
+</head>
+<body>
+    <div class="one">
+        <form>
+       <h3>First Name</h3> <input type="text">
+       <h3>Last Name</h3> <input type="text">
+       <h3>Nhập Email</h3> <input type="email"> 
+       <h3>Nhập số Phone</h3> <input type="number"> <br>
+       <button>submit</button>
+        </form>
```

Comment:
We should re-format this html file again, it should have the right indentation. (căn chỉnh lại các thẻ HTML cho đúng nha, code thể này nhìn loạn quá, thụt đầu hàng cho chuẩn)